tests/unit/test_documents_dojson.py

```diff
@@ -436,6 +436,7 @@ def test_marc21_to_publishers_publicationDate():
     marc21json = create_record(marc21xml)
     data = marc21tojson.do(marc21json)
     assert not data.get('publishers')
+    assert data.get('publishers') != []
```

Comment:
I think you can replace asserts : 
```
assert not data.get('publishers')
assert data.get('publishers') != []
```

by:
```
assert "publishers " not in data
```
It should be enough to test that publishers key does not exist.

Reply:
But it's a detail

Reply:
Ok, corrected. Seems to work, indeed.